currency-converter/src/components/Dropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import Select, { StylesConfig } from 'react-select';
+
+interface IDropdown {
+  label: string;
+  options: { value: string }[];
+}
+
+export default function Dropdown({ label, options }: IDropdown) {
+  const customStyles: StylesConfig = {
+    control: (base, state) => ({
+      ...base,
+      borderRadius: '0.375rem',
+      borderColor: state.isFocused ? '#6B7280' : '#e5e7eb',
+      padding: '0.1rem',
+      boxShadow: 'none',
+      '&:hover': {
+        borderColor: 'none',
+      },
+      '&:focus-visible': {
+        outline: 'none',
+        boxShadow: '0 0 0 2px gray',
+      },
+    }),
+    menu: (base) => ({
+      ...base,
+      backgroundColor: 'white',
+      boxShadow: '0 2px 10px rgba(0, 0, 0, 0.1)',
+    }),
+    option: (base, state) => ({
+      ...base,
+      padding: '0.75rem',
+      fontSize: '0.875rem',
+      cursor: 'pointer',
+      backgroundColor: state.isFocused ? '#F3F4F6' : 'white',
+      color: '#1F2937',
+      '&:active': {
+        backgroundColor: '#6B7280',
+      },
+    }),
+    placeholder: (base) => ({
+      ...base,
+      color: '#6B7280',
+      fontSize: '0.875rem',
+    }),
+    indicatorSeparator: (base) => ({
+      ...base,
+      display: 'none',
+    }),
+  };
```

Comment:
Let's move this `const` outside the the react component. Why? Because whenever this component re-renders, it will re-create this `const` which is not optimized. We should always move those `const` and `functions` which are not dependent on the components states etc.